metrics/src/test/java/io/opencensus/metrics/TimeSeriesTest.java

```diff
@@ -104,28 +100,6 @@ public void create_WithNullPoint() {
     TimeSeriesCumulative.create(Collections.<LabelValue>emptyList(), points, TIMESTAMP_1);
   }
 
```

Comment:
It might be clearer to split this test suite now that the two TimeSeries classes are in separate files.

Reply:
Done.